### ISSUE TITLE
The behavior of sphere parameters gone wrong

### ISSUE THREAD

**Issue by oligami** [mitsuba-renderer/mitsuba2#145]
I've updated mitsuba 2 today and noticed that parameters calculation of sphere has gone wrong. The position of sphere's center is multiplied by its radius!
Documents say that the two declarations below are equivalent. However, the sphere was rendered at [2, 0, 0] in the image obtained by the lower declaration.
```
<shape type="sphere">
    <transform name="to_world">
        <scale value="2"/>
        <translate x="1" y="0" z="0"/>
    </transform>
    <bsdf type="diffuse"/>
</shape>

<shape type="sphere">
    <point name="center" x="1" y="0" z="0"/>
    <float name="radius" value="2"/>
    <bsdf type="diffuse"/>
</shape>
```

**Comment by Speierers** [comment]
Hi @oligami ,

Thanks for reporting this. It should be fixed now (7bf677fa)

**Comment by oligami** [comment]
Thanks!
